.github/workflows/gardener_issue_comment.yml

```diff
@@ -89,7 +89,7 @@ jobs:
             echo "Found issue on Gardener board. Current issue status is: '${current_status}'"
           fi
 
-          if [ "$current_status" = "Blocked / Waiting" ] ; then
+          if [ "$current_status" = "\"Blocked / Waiting\"" ] ; then
```

Comment:
Alternatively you can add `-r` to the `jq` command above to have it strip the quotes.

Reply:
Nice, I like that better.